### ISSUE TITLE
Subtitle in header error: type should not be null

### ISSUE THREAD

**Issue by stueynet** [GeekyAnts/NativeBase#413]
Trying to use Subtitle in the Header component. Looks like it should be fine to just include but I get an error:

```
                      <Header>
                        <Button transparent onPress={() => this.popRoute()}>
                            <Icon name="ios-arrow-back"/>
                        </Button>

                        <Title>Main title</Title>
                        <Subtitle>Testing</Subtitle>

                        <Button transparent onPress={() => this.toggleDrawer()}>
                            <Icon name="ios-menu"/>
                        </Button>
                    </Header>
```

Error is: 
Warning: React.createElement: type should not be null, undefined, boolean, or number.

**Comment by sankhadeeproy007** [comment]
Fixed with the above commit.

**Comment by stueynet** [comment]
Thanks @sankhadeeproy007. I see you re-opened this. Can you explain?

**Comment by sankhadeeproy007** [comment]
There's a problem with the way `Subtitle` is being exported. Will be fixed today.

**Comment by stueynet** [comment]
Hey guys. Just wanted to follow up on this. Can you let me know what the problem and fix is and we can fork it?

**Comment by stueynet** [comment]
Guys we would be happy to fork and fix this issue if you just let me know what the problem is.

**Comment by sankhadeeproy007** [comment]
Sorry, we've been a bit busy with the development of NativeBase 2.0. The problem here seems to be that `Subtitle` isn't being exported [here](https://github.com/GeekyAnts/NativeBase/blob/master/index.js). As you would see, the import is correct for the subtitle, however it isn't being exported. Should be a very easy fix.

**Comment by sanketsahusoft** [comment]
Fixed with 2.0. Close it if you've moved to 2.0.
